.eslintrc.next.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
               1, // common for i + 1 in a loop
               2, // many .slice(2) since the '0x' prefix should be removed while calling 3rd-party library
               16, // toString(16)
+              30000, // rpc request timeout: 30 seconds
```

Comment:
The number 30000 may not be common in the project. Should we name it directly?

```js
const DEFAULT_RPC_TIMEOUT = 30000;
```

Reply:
Done. Also updated `method.ts` to use `DEFAULT_RPC_TIMEOUT` .